docs/deployed-contracts/index.md

```diff
@@ -163,8 +169,10 @@ pagination_next: deployed-contracts/goerli
 
 
 - **Tooling contracts:**
-  - AllowedRecipientsBuilder: [`0x958e0D946D014F377421a53AB5f9180d4485e63B`](https://etherscan.io/address/0x958e0D946D014F377421a53AB5f9180d4485e63B)
-  - AllowedRecipientsFactory: [`0x83E976758B7AB1bb676A4fEA073Fa0E2A807642B`](https://etherscan.io/address/0x83E976758B7AB1bb676A4fEA073Fa0E2A807642B)
+  - AllowedRecipientsBuilder (Proposed to remove): [`0x958e0D946D014F377421a53AB5f9180d4485e63B`](https://etherscan.io/address/0x958e0D946D014F377421a53AB5f9180d4485e63B)
```

Comment:
I think the old builder and factory should be left in case there is a need to deploy a setup with one token